blogs.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  <meta charset="UTF-8">
+  <title>Blog</title>
+</head>
+<body>
+  <header>
+    <div class="flexbox">
+      <div class="left-side">
+        <img src="adopt-a-dog/adoptadog-logo.svg" alt="logo" width="600">
+        <h1>Seattle’s premiere dog adoption agency</h1>
+      </div>
+      <div class="right-side">
+        <img class="cart" src="adopt-a-dog/cart-icon.svg" alt="cart image" width="50" height="50">
+        $1234.56
+        <br>
+        <button type="button">Checkout</button>
+      </div>
+    </div>
+  </header>
+  <div class="nav-container">
+    <nav>
+      <a href="index.html">Home</a> /
+      <a href="dogs.html">Dogs</a> /
+      <a href="blogs.html">Blog</a>
+    </nav>
+  </div>
+  <main>
+    <h2> Adoptadog Blog </h2>
+    <div class="flex-container2">
+      <div class="left-column2">
+        <img src="adopt-a-dog/images/patrick-hendry-221863-unsplash.jpg" alt="Image of a man holding a dog" width="400" height="350">
```

Comment:
You'll want to group each of these images in with the blog post. You don't want to use two columns and try to aligh the text in one column with the images in another. Something like

<article class="dog-blog">
  <img>
  <h3>Headline</h3>
  <p>text</p>
</article>

You'll then move the image around.